newsfragments/2688.misc.rst

```diff
@@ -0,0 +1 @@
+Ursula will now check for active stakes on startup.
```

Comment:
wdyt about this as a feature newsfragment instead? Feels like it should be more prominent since nodes will not start. Just food for thought...